pulp_ansible/app/tasks/utils.py

```diff
@@ -7,11 +7,12 @@
 from yaml.error import YAMLError
 
 
-def get_page_url(url, page=1):
+def get_page_url(url, page_size, page=1):
     """Get URL page."""
     parsed_url = urlparse(url)
     new_query = parse_qs(parsed_url.query)
     new_query["page"] = page
+    new_query["page_size"] = page_size
```

Comment:
@bmbouter bugfix is here, adding `?page_size=PAGE_SIZE` on galaxy requests
https://pulp.plan.io/issues/5494#note-1

Reply:
So when syncing from another Pulp serving the galaxy API, what would happen if page_size was unset? I guess I don't fully understand the bug claim itself.

Reply:
actually I mixed things, the bug is fixed just by making `PAGE_SIZE`constant equals to 100. As it is the [default value for pulp](https://github.com/pulp/pulpcore/blob/master/pulpcore/app/settings.py#L129). But it would break when syncing from galaxy, as its default is 10. So we are appending `?page_size=100`.
Making both pulp and galaxy default to 100. And if the user wants, giving them power to parameterize it.